vision/google/cloud/vision/_gax.py

```diff
@@ -25,10 +26,14 @@ class _GAPICVisionAPI(object):
 
     :type client: :class:`~google.cloud.vision.client.Client`
     :param client: Instance of ``Client`` object.
+
+    :type kwargs: dict
+    :param kwargs: Additional keyword arguments are sent to the GAPIC client.
     """
-    def __init__(self, client=None):
+    def __init__(self, client=None, credentials=None):
         self._client = client
-        self._annotator_client = image_annotator_client.ImageAnnotatorClient()
+        self._annotator_client = image_annotator_client.ImageAnnotatorClient(
+            credentials=credentials, lib_name='gccl', lib_version=__version__)
```

Comment:
I think this is where things should change.

I think you want `client.credentials` instead of the passed in credentials that you added.

You can see [here](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/2c608f95d7d5db2cbfdce77770f523a489b513f4/vision/google/cloud/vision/client.py#L65) that the `credentials` can be passed into the client on initialization which is the pattern the other services use.